### ISSUE TITLE
Add informations about project on the main page

### ISSUE THREAD

**Issue by WTobor** [DevAdventCalendar/DevAdventCalendar#307]
Add informations about project and its rules on the main page:

- programistyczny kalendarz adwentowy online
- każdego dnia otwiera się nowe okienko
- odpowiadasz na pytanie i zbierasz punkty
- osoby z największą liczbą punktów zdobywają nagrody

**Comment by kokosanki** [comment]
Hi, does it still need to be implemented? I can see that similar information are already on the page

**Comment by WTobor** [comment]
Yes - we need to decide how to describe it and show it always on the website. Maybe we will add a new picture?

**Comment by kokosanki** [comment]
I guess that would work
